src/server/views/lists/partials/lawyers/macros.njk

```diff
@@ -82,7 +82,7 @@
       <strong>Telephone</strong>: <a class="govuk-link" href="tel:{{ item.jsonData.phoneNumber }}">{{ item.jsonData.phoneNumber }}</a>
     </p>
   {% endif %}
-  {% if item.jsonData.emergencyPhoneNumber %}
```

Comment:
is this correct?  BaseDeserialisedWebhookData emergencyPhoneNumber but not contactPhoneNumber

Reply:
Ah, good spot. I just did a query on the database to see what was missing or not, and did it based on that. This user had emergencyPhone missing (so I assumed contactNumber should be rendered). Perhaps the migration didn’t execute properly? I can rerun the migration